vm/src/stdlib/itertools.rs

```diff
@@ -174,14 +174,13 @@ mod decl {
     #[pyclass(name = "count")]
     #[derive(Debug, PyPayload)]
     struct PyItertoolsCount {
-        cur: PyRwLock<BigInt>,
-        step: BigInt,
+        cur: PyRwLock<PyObjectRef>,
+        step: Option<PyIntRef>,
     }
 
     #[derive(FromArgs)]
     struct CountNewArgs {
-        #[pyarg(positional, optional)]
-        start: OptionalArg<PyIntRef>,
```

Comment:
`#[pyarg]` and `OptinoalArg` is not expected to be removed.
(See the test results how this change broke tests)

Use `OptionalArg<PyObjectRef>`